packages/app-layout/test/app-layout.test.js

```diff
@@ -274,6 +275,18 @@ describe('vaadin-app-layout', () => {
         layout.drawerOpened = true;
         expect(toggle.getAttribute('aria-expanded')).to.equal('true');
       });
+
+      describe('a11y', () => {
+        beforeEach(async () => {
+          layout.drawerOpened = true;
+          await nextFrame();
+        });
+
+        it('should not close drawer on pressing Escape', () => {
+          escKeyDown(layout);
```

Comment:
I found that the component already has some tests to verify the ability to close the drawer on <kbd>Escape</kbd>. I'm now under the impression that the feature is already there. Does it not work? 🤔 

https://github.com/vaadin/web-components/blob/3d1e8a146415cace4e939be000be52aee2967191/packages/app-layout/test/app-layout.test.js#L261-L264

https://github.com/vaadin/web-components/blob/3d1e8a146415cace4e939be000be52aee2967191/packages/app-layout/test/app-layout.test.js#L367-L370

Reply:
Good catch. I will take a closer look to see when it was added.

Reply:
It was added here https://github.com/vaadin/web-components/pull/2395

Reply:
Related tests and logic can be removed, see my comment below.